src/main/java/com/ibm/as400/access/list/SpooledFileOpenList.java

```diff
@@ -765,25 +771,25 @@ private final int findFieldLength(int field, int vrm)
             case JOB_NUMBER: return 6;
             case NAME: return 10;
             case NUMBER: return 4;
-            case STATUS: return format_ == 3 ? 4 : 10;
+            case STATUS: return (format_ == 3 || format_ == 4) ? 4 : 10;
```

Comment:
Since you're touching this part, a cleanup request: Please add a `TODO` tag to the code to replace the integer literals used for the `format_` field with a named constant or something sensible

Reply:
https://github.com/IBM/JTOpen/pull/29/commits/13500c6828d1c0f88f5afd30c7b47d8e49a28779